src/Couchbase.Lite.Shared/Store/SymmetricKey.cs

```diff
@@ -251,7 +251,7 @@ public Stream DecryptStream(Stream stream)
         /// </summary>
         /// <returns>The stream to write to for encryption</returns>
         /// <param name="baseStream">The stream to read from</param>
-        public CryptoStream CreateStream(Stream baseStream)
+        public Stream CreateStream(Stream baseStream)
```

Comment:
With our AES CTR implementation, we need to return something that isn't a CryptoStream.  Using the more generic Stream gives us that flexibility.